<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAbstractNavigation</class>
 <widget class="QAbstractInteractorStyle" name="QAbstractNavigation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>40</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QAbstractNavigation</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sliceSpinBoxX">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="sliceHorizontalSliderX">
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sliceSpinBoxY">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="sliceHorizontalSliderY">
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sliceSpinBoxZ">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="sliceHorizontalSliderZ">
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAbstractInteractorStyle</class>
   <extends>QWidget</extends>
   <header>qabstractinteractorstyle.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sliceSpinBoxX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliceHorizontalSliderX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliceHorizontalSliderX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliceSpinBoxX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliceSpinBoxY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliceHorizontalSliderY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliceHorizontalSliderY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliceSpinBoxY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliceSpinBoxZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliceHorizontalSliderZ</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliceHorizontalSliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliceSpinBoxZ</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
